src/tasks/init.js

```diff
@@ -22,18 +22,22 @@ const path = require('path');
 const files = require('./constants/files');
 
 module.exports = () => {
-  [
-    files.CONTAINER_FILENAME,
-    files.LIB_SANDBOX_HTML_FILENAME
-  ].forEach((filename) => {
-    fs.copy(
-      path.resolve(files.CLIENT_SRC_PATH, filename),
-      path.resolve(files.CONSUMER_CLIENT_SRC_PATH, filename),
-      {
-        clobber: false
-      }
-    );
-  });
+  const descriptor = require(path.resolve(files.EXTENSION_DESCRIPTOR_FILENAME));
+
+  if (descriptor.platform === 'web') {
```

Comment:
The readme looks good. I would say that if they try to run `init` and their extension is not a web extension, we should log an error and bail with an non-zero exit code.